config/routes.rb

```diff
@@ -1,34 +1,37 @@
 Rails.application.routes.draw do
-  get 'home', to: 'home#index'
-  get 'pricing', to: 'home#pricing'
+  get 'home',     to: 'home#index'
+  get 'pricing',  to: 'home#pricing'
   get 'about-us', to: 'home#about_us'
   get 'contacts', to: 'home#contacts'
 
   get "/404", to: "errors#not_found"
   get "/422", to: "errors#unacceptable"
   get "/500", to: "errors#internal_error"
 
+  match :generate_token, via: %i[post patch], to: 'tokens#generate_token'
```

Comment:
move to "profile" as member methods. But keep this controller.

basically I want to have routes like

/account/profile/generate_token
/account/profile/deactivate_token

Reply:
mb in more CRUD-like style?
```
resource :profile do
  resource :token, only: [:create, destroy]
end
```
?